README.md

```diff
@@ -1,10 +1,10 @@
 ## RGeo::ActiveRecord
 
 [![Gem Version](https://badge.fury.io/rb/rgeo-activerecord.svg)](http://badge.fury.io/rb/rgeo-activerecord)
-[![Build Status](https://travis-ci.org/rgeo/rgeo-activerecord.svg?branch=master)](https://travis-ci.org/rgeo/rgeo-activerecord)
+[![Status](https://github.com/rgeo/rgeo-activerecord/workflows/Tests/badge.svg?branch=v7-dev)](https://github.com/rgeo/rgeo-activerecord/actions)
```

Comment:
In fast-polyline i found a way to have badge with the same colors: i'm always based on shield.io:

```markdown
[![Gem version](https://img.shields.io/gem/v/rgeo-activerecord)](https://rubygems.org/gems/rgeo-activerecord)
[![GitHub Workflow Status](https://img.shields.io/github/workflow/status/rgeo/rgeo-activerecord/Tests)](https://github.com/rgeo/rgeo-activerecord/actions?query=workflow%3A%22Tests%22)
```

<sub> this is not tested, just copied and adapted from [this readme](https://github.com/klaxit/fast-polylines#fast-polylines) </sub>

Reply:
It worked thanks!